integration/obscuroscan/obscuroscan_test.go

```diff
@@ -55,7 +55,7 @@ func TestObscuroscan(t *testing.T) {
 	require.NoError(t, err)
 
 	// wait for the msg bus contract to be deployed
-	time.Sleep(5 * time.Second)
+	time.Sleep(10 * time.Second)
```

Comment:
Was sleep for 5 secs too low before?

Reply:
Yeah.. I thought this would fix it, but still makes the tests flaky. Ended up skipping both tests for now.